public/src/components/PottyForm.jsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
-export const PottyForm = (props) => {
-  const { children } = props;
-  return (
-    <form className="mainform">
-      <h1>Record Incident</h1>
-      {children}
-    </form>
-  );
-};
+const StyledForm = styled.form`
+  max-width: 250px;
```

Comment:
You're gonna want to break yourself of relying on arbitrary pixel widths. There's a smattering of them throughout this commit that has me uneasy

Reply:
The alternative being setting sass variables?

Reply:
It's difficult to maintain arbitrary pixel widths throughout a system. There were several random font sizes, icon sizes, etc throughout the component definitions. If you hoist those into a theme you have a centralized repository of "go to" dimensions such as font ramping, max content widths, etc.

Reply:
Got it, I'll give it a shot

Reply:
I'm not sure to what extent things should go into the theme, so I'm going to err on the side of "everything" and you can tell me what's unnecessary next commit.

Reply:
Ok I think I've extracted all the fairly common stuff, and left all the "unique to this specific component or type of component". You can tell me whether I went too far or not far enough.

Reply:
Yep, you've done a decent job from what I see. You can then do things like:

```
export const Button = styled.button`
   ${({ theme }) => `
      ${theme.button}
   `
```

Seems silly, right? Until you realize you can also do:

```jsx
import Button from './Button';

export default () => <Button as="a">I'm a button looking thing but actually an anchor</Button>
```

Our styled system is basically a giant collection of component styles + some additional palette properties, and the components themselves are just the right HTML element with the theme reference inside of it

Reply:
Gotcha. I've extracted more of the theme, although I left the spacing cuz I'm not doing a too much in terms of configurable spacing for this project.